### DISCUSSION TITLE
skoala/best-practice/use-skoala-01/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1510] [Announcements]
# skoala/best-practice/use-skoala-01/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/best-practice/use-skoala-01/

**Comment by gssjl2008** [comment]
代码在截图里，是否可以贴出来

**Reply by samzong** [reply]
文档中代码相关部分均已支持一键复制了，您指的是哪部分呢？建议截图或者描述对应具体的章节。

**Comment by liyuerich** [comment]
opentelemetry-demo 应用，没有找到这个应用

**Reply by samzong** [reply]
https://docs.daocloud.io/skoala/best-practice/use-skoala-01/#_2
![20230524190138](https://github.com/DaoCloud/DaoCloud-docs/assets/13782141/2c58d49f-08b9-4636-b437-d85aee6abdfb)

**Reply by gssjl2008** [reply]
直接引用安装命令到 文档内，而不是让用户跳转到 github 自己找，相对而言，更友好吧。

**Reply by samzong** [reply]
> 直接引用安装命令到 文档内，而不是让用户跳转到 github 自己找，相对而言，更友好吧。

文档要考虑以一个合理的方式；由于应用更新和迭代比较快，如果没有及时文档，反而会带来更多错误的问题哈。

**Reply by Michelle951** [reply]
improved: https://docs.daocloud.io/skoala/best-practice/use-skoala-01/
